core/src/test/java/org/commonjava/indy/fixture/MockContentAdvisor.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2013 Red Hat, Inc. (jdcasey@commonjava.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.fixture;
+
+import org.commonjava.indy.spi.pkg.ContentAdvisor;
+import org.commonjava.indy.spi.pkg.ContentQuality;
+import org.commonjava.maven.atlas.ident.util.ArtifactPathInfo;
+
+import javax.enterprise.inject.Default;
+
+//TODO: This provider is fully duplicated with org.commonjava.indy.test.fixture.core.MockContentAdvisor in
+//      test/fixture-core for some dependency reasons in addons. Should be refactored in another common
+//      test module in the future.
+
+@Default
+public class MockContentAdvisor
+        implements ContentAdvisor
+{
+    @Override
```

Comment:
It's right to use a pre-defined list to replace this Maven-specific imp. But I'm not sure what's an exact list for this path. As it's only serve for maven now, I just simply use a snapshot check here as it is used before in DefaultDownloadManager. 
@jdcasey do you have any ideas or examples on this list?